client-js/app/elements/app-link.ts

```diff
@@ -0,0 +1,30 @@
+const MOUSE_MAIN_BUTTON = 0;
+
+@component('app-link')
+@extend('a')
+export class AppLink extends polymer.Base {
+
+  @property({type: Array})
+  path: string[];
+
+  isSpecialClick(e): boolean {
+    return (e.ctrlKey || e.metaKey || e.button !== MOUSE_MAIN_BUTTON);
+  }
+
+  @listen('click')
+  onClick(e) {
+    if (this.isSpecialClick(e)) return;
```

Comment:
I personally prefer braces for consistency, and if you include braces, I prefer not to one-line them.

```typescript
if (this.isSpecialClick(e)) {
  return;
}```

Everyone is different on this though.